src/testing/reduced_order_pod_adaptation.cpp

```diff
@@ -30,13 +33,12 @@ int ReducedOrderPODAdaptation<dim, nstate>::run_test() const
     const Parameters::AllParameters param = *(TestsBase::all_parameters);
 
     pcout << "Running Burgers Rewienski with parameter a: "
-          << param.reduced_order_param.rewienski_a
+          << param.burgers_param.rewienski_a
           << " and parameter b: "
-          << param.reduced_order_param.rewienski_b
+          << param.burgers_param.rewienski_b
           << std::endl;
 
-    using Triangulation = dealii::Triangulation<dim>;
-    std::shared_ptr<Triangulation> grid = std::make_shared<Triangulation>();
+    std::shared_ptr<dealii::Triangulation<dim>> grid = std::make_shared<dealii::Triangulation<dim>>();
```

Comment:
Maybe you should use `Triangulation` as before and use `#if` statements at the top for if `PHILIP_DIM==1` etc, just to avoid a possible bug when someone eventually extends this to multidimensional problems

Reply:
This is just a test specific to Burgers Rewienski which is probably only going to ever be a 1D case anyway, so it should be fine.